src/pool/srv_pool.c

```diff
@@ -3681,6 +3715,11 @@ ds_pool_create_handler(crt_rpc_t *rpc)
 	ABT_rwlock_wrlock(svc->ps_lock);
 	ds_cont_wrlock_metadata(svc->ps_cont_svc);
 
+	if (svc->ps_error != 0) {
+		rc = svc->ps_error;
+		goto out_tx;
```

Comment:
NIT
```suggestion
		D_GOTO(out_tx, rc);
```

Reply:
@knard38, this conversion to `goto` is intentional; as I understand we want to eventually remove `D_GOTO` (personally, I'm not sure that'll ever happen naturally).

Reply:
Thanks for the information, I was not aware of that.